src/converter.ts

```diff
@@ -294,6 +303,17 @@ export class Converter {
 
     const ret = file.convert(this.options.output, { extension: 'pdf' })
 
+    // Generate PDF
+    const browser = await browserManager.browserForConversion()
+
+    if (browser.kind === 'firefox' && !this._firefoxPDFConversionWarning) {
+      this._firefoxPDFConversionWarning = true
+
+      warn(
+        'Using Firefox to convert Markdown to PDF: The output may include some incompatible renderings compared to the PDF generated by Chrome.'
```

Comment:
Currently, PDF conversion by Firefox has some incompatible rendeings compared to PDF generated by Chrome. e.g. There are some vanished elements in PDF, such as the background gradient like `gaia` theme, the element with CSS filter, etc.

:arrow_right: Should treat PDF conversion backed by Firefox as fallback